src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -496,6 +510,1759 @@ def evaluate(self, dataset, metric='auto', verbose=True, batch_size=64):
         """
         Evaluate the model by making predictions of target values and comparing
         these to actual values.
+        
+        
+# Regression model names.
+
```

Comment:
Whitespace issues here.

Reply:
I'm not sure what this is. It looks like this is just copied from logistic_classifier.py. You don't want to copy any code from that file. You just want to pass the new parameters to [the create call for logistic regression](https://github.com/apple/turicreate/blob/86c65447504be4e4bdd3378d88957c6f1b0d4d63/src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py#L164).

Reply:
Yes, it's copied from logistic_classifier.py.